Description: Add debugging comments to cdctl.h.
 Patch created for some version between 1.0beta1-5 and and 1.0beta1-8.
 Converted to quilt for 1.0beta-12.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-07-29

--- a/cdctl.h
+++ b/cdctl.h
@@ -213,9 +213,19 @@
                    start = cd_trklist[status_track].track_start;
                    stop = start + cd_trklist[status_track].track_len - _CDCTL_SENSITIVITY;
                    now = ((sc.cdsc_absaddr.msf.minute) * 60 + sc.cdsc_absaddr.msf.second) * 75 + sc.cdsc_absaddr.msf.frame - CD_MSF_OFFSET;
+/*
+cout << "=============" << endl;
+cout << "start:" << start << endl;
+cout << "stop:" << stop << endl;
+cout << "now:" << now << endl;
+*/
 	           if(now>0 && (now<start || now>=stop)){
                       status_state=ssPlaying;
                       selecttrack();
+/*
+cout << "Play track " << status_track << endl;
+cout << "=============" << endl;
+*/
                       doStatus();
                       return;
                    }
